app/models/chapter.py

```
@@ -0,0 +1,11 @@
+from sqlalchemy import Column, Integer, String, Text, TIMESTAMP
```

Ruff (0.8.2)

`sqlalchemy.TIMESTAMP` imported but unused

Remove unused import: `sqlalchemy.TIMESTAMP`

(F401)